### DISCUSSION TITLE
How to reseed a file that has been moved and renamed？

### DISCUSSION THREAD

**Discussion by VeroFess** [transmission/transmission#6316] [Q&A] [answered]
I am currently using the Transmission suite on Synology with Radarr. However, my storage volume is not located in the same partition as the download directory, thus I am unable to organize the files using symbolic links.

My question is, if I have the original magnet link, is there a way for me to locate the renamed file and reseed it?

**Comment by Pentaphon** [comment]
If you need to seed from the original magnet link and thus torrent file, your file needs to be reverted back to its original filename. You can't re-seed anything unless it is the same file and the same filename. That's how it works for all torrent clients.

**Reply by ghost** [reply]
I don't understand this answer. I don't use magnet links, but when I download a torrent and then rename the files inside the torrent using transmission remote GUI the torrent continues to seed and upload back to the community no problem.

**Reply by Pentaphon** [reply]
Sure. As long as you can obtain the torrent and original files it will seed. Where is the confusion? The magnet link is just a way to get the torrent file from another client via DHT or trackers and once your torrent file loads into your client, it will verify itself and seed.

**Reply by ghost** [reply]
The confusion is your statement about reverting back to the original filenames. You don't need to revert the files back to their originals. As long as you rename them inside the torrent file once you've loaded it into transmission, it will seed them just fine.

**Comment by killemov** [comment, marked as answer]
> my storage volume is not located in the same partition as the download directory, thus I am unable to organize the files using symbolic links.

No! You CAN use symbolic links just not hard inks. You can even use symbolic links over network shares.

If the torrent is (almost) dead you will have a hard time getting the magnet resolved through other clients. Better go look for the torrent yourself based on its hash.

You then have to decide where the optimal target location of your payload is going to be. (centralized or current payload partition) And then you need to (hard)link and/or rename the files back in its original format.
